docs/develop/go/message-passing.mdx

```diff
@@ -334,8 +334,8 @@ An Update is a synchronous, blocking call that can change Workflow state, contro
 
 A Client sending an Update must wait until the Server delivers the Update to a Worker.
 Workers must be available and responsive.
+Setting a timeout with a context provides a hard limit on how long a client will wait for a response.
```

Comment:
This was my addition to make sure users think about setting a timeout.

Reply:
I'm not super experienced in the other SDKs to know where/how to put that there, too. Anyway, something we can always do later.

Reply:
I like that; nice addition.